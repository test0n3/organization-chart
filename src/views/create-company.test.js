import React from "react";
import { render } from "@testing-library/react";
import CompanyName from './create-company';

test("Create Company component with CSS ", () => {
  const { asFragment } = render(<CompanyName />);

  expect(asFragment()).toMatchSnapshot();
});
