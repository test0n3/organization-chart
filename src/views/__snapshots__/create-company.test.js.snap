// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Company component with CSS  1`] = `
<DocumentFragment>
  <div
    class="css-kwio78-CompanyName"
  >
    <form
      class="css-1s4v5s4-InputCompanyName"
    >
      <input
        class="css-1rqyb0g-InputCompanyName"
        name="company"
        placeholder="Company's name"
        type="text"
      />
    </form>
  </div>
</DocumentFragment>
`;
